modules/nf-core/pretextsnapshot/main.nf

```diff
@@ -0,0 +1,31 @@
+process PRETEXTSNAPSHOT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::pretextsnapshot=0.0.4" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pretextsnapshot:0.0.4--h7d875b9_0':
+        'quay.io/biocontainers/pretextsnapshot:0.0.4--h7d875b9_0' }"
+
+    input:
+    tuple val(meta), path(pretext)
+
+    output:
+    tuple val(meta), path("*.png"), emit: picture
```

Comment:
It looks like a user can specify multiple types of output, you need to capture all these here, or have separate channels for all [^1]:

```
	-f/--format fmt:        Image format, one of "png"(default) "bmp" or "jpeg".
```

[^1]: https://nf-co.re/docs/contributing/modules#inputoutput-options Point 1.

Reply:
technically you can only specify one (so there shouldn't be png and jpeg), but I changed the output glob to allow for all possible types